src/Github/Api/V3/CreateReleaseThroughApiCall.php

```diff
@@ -60,7 +60,7 @@ public function __invoke(
 
         $response = $this->client->sendRequest($request);
 
-        Psl\invariant($response->getStatusCode() >= 200 || $response->getStatusCode() <= 299, 'Failed to create release through GitHub API.');
+        Psl\invariant($response->getStatusCode() >= 200 && $response->getStatusCode() <= 299, 'Failed to create release through GitHub API.');
```

Comment:
Fine by me, but we didn't catch this during refactoring due to missing tests: we can haz?

Reply:
u now haz 😊